src/endpoints/account-addresses.js

```diff
@@ -0,0 +1,63 @@
+import { singularize } from 'inflected'
+
+import BaseExtend from '../extends/base'
+
+class AccountAddressesEndpoint extends BaseExtend {
+  constructor(endpoint) {
+    super(endpoint)
+
+    this.endpoint = 'addresses'
+  }
+
+  All({ account, token = null }) {
```

Comment:
If you extend `CRUDExtend` class you wouldn't need to write any method in this class. `CRUDExtend` and `BaseExtend` that it inherits already have all the default implementations for these methods. Take a look at `base.js` and `crud.js`

Reply:
I assume I can't use the default implementations inside CRUDExtend because my account addresses resource is parameterized by account ID, and it the form of `accounts/{accountId}/addresses/{addressId}`, so CRUDExtend doesn't support it

Reply:
I see, good point 👍